src/components/Button.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Button = ({ buttonName }) => <div className="button">{buttonName}</div>;
+
+Button.propTypes = {
+  buttonName: PropTypes.string.isRequired,
+};
+
+export default Button;
```

Comment:
- this component should accept `color` and `wide` as an incoming props 
- ensure `wide` is boolean via propTypes [ see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/WideProps.png) ]
- ensure default `color` prop value is ORANGE (or hex orange-like codes if you like. Ensure this with `defaultProps` [ see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/colorProps.png) ]